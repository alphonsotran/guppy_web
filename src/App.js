import React from 'react';
import Form from './components/Form';

import { AppWrapper, Header } from './styles';

function App() {
  return (
    <AppWrapper>
      <Header>Guppy: A Url Shortener 🐟</Header>
      <Form />
    </AppWrapper>
  );
}

export default App;
