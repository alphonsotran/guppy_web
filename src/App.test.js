import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

it('expects to render App component', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
