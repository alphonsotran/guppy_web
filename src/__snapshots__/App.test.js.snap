// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render App component 1`] = `
<styled.div>
  <styled.h1>
    Guppy: A Url Shortener 🐟
  </styled.h1>
  <Form />
</styled.div>
`;
